omod/src/main/java/org/openmrs/module/fhir/providers/RestfulPersonResourceProvider.java

```diff
@@ -127,9 +127,9 @@ public MethodOutcome createFHIRPerson(@ResourceParam Person person) {
 	public MethodOutcome updatePersonConditional(@ResourceParam Person thePerson, @IdParam IdDt theId) {
 		MethodOutcome retVal = new MethodOutcome();
 		OperationOutcome outcome = new OperationOutcome();
-		try {
-			Person person = personResource.updateFHIRPerson(thePerson, theId.getIdPart());
-		} catch (Exception e) {
+		//try {
```

Comment:
@surangak  , I removed this generic exception. Now the PersonServiceImpl.java file will validate the request body in fine grained level and give a more meaningful message.